### DISCUSSION TITLE
Offer Win10to7 on GitHub or anywhere other than DeviantArt

### DISCUSSION THREAD

**Discussion by TheSystemGuy1337** [ImSwordQueen/Win10to7Issues#157] [General]
### Before sending your suggestion

- [ ] My version of Windows is up to date (Atleast up to April 2024 as per update warning)
- [ ] I'm running the latest version of the Windows 10to7 Transformation Pack.

### What version of Windows 10 are you using the transformation pack on?

NOT APPLICABLE

### What would you like to suggest for 21H2to7?

Host downloads for Win10to7 on GitHub rather than DeviantArt, as DA requires an account to download files. I didn't download this yet, so all fields are blank and those that are required are marked as N/A

### Additional Context

DeviantArt will eventually gatekeep URLs from guests. They already took the first step towards gatekeeping.

**Comment by TheBinaryShadow** [comment]
I went trough 2FA sign-in at 2AM just to say, please, host it on GitHub

**Comment by TheSystemGuy1337** [comment]
Also, on top of that; we should add the Windows Classic theme.

**Comment by alias-old** [comment]
you aren't supposed to use the download button, the download link is on the deviantart description

**Comment by TheSystemGuy1337** [comment]
> you aren't supposed to use the download button, the download link is on the deviantart description

I still believe the download should be hosted on GitHub rather than DA.

**Comment by ImSwordQueen** [comment]
![image](https://github.com/user-attachments/assets/46db3c06-1a82-43e9-94a5-7844a241558c)

**Comment by TheSystemGuy1337** [comment]
We should still host it on GitHub. Revert8Plus is hosted on GitHub, why can’t Win10to7? And it’s only a matter of time before DA decides to gatekeep URLs from guests and require users to sign up just to even view and click on them.

**Comment by maphapthieunu** [comment]
> We should still host it on GitHub. Revert8Plus is hosted on GitHub, why can’t Win10to7? And it’s only a matter of time before DA decides to gatekeep URLs from guests and require users to sign up just to even view and click on them.

Could you click on the link in the warning message?

**Comment by Victor35345-afk** [comment]
why do you even want to host it on github?

**Comment by holdentheplanet** [comment]
Bump. Why is this hosted on DeviantArt of all places, through a mega link and closed source? I'm still not entirely convinced this is clean software because of this. If autokms can stay up on here without problems surely this can too.

**Reply by ImSwordQueen** [reply]
Deviantart is (was) the place where people published their themes/customization stuff for Windows.
I use mega instead of the DA download because DA requires you to log in to download stuff and also doing it on mega makes it easy to upload only the exe and hash.
Also it is closed source because I do not trust people enough to not take my work. remove all credits or change the ui slightly and resell it as their own work. Revert8Plus isn't open source either (albeit it is hosted on github releases instead of being offsite)
